Releases/LatestRelease/StixParser.md

```diff
@@ -0,0 +1 @@
+- convert to package
```

Comment:
no need to say that in release notes

Reply:
buy something should be there. don't worry, itllbe fixed